packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -41,6 +42,10 @@ export async function beaconHandler(args: IBeaconArgs & IGlobalArgs): Promise<vo
   const options = beaconNodeOptions.getWithDefaults();
 
   const abortController = new AbortController();
+  // We set infinity to prevent MaxListenersExceededWarning which get logged when listeners > 10
+  // Since it is perfectly fine to have listeners > 10
+  setMaxListeners(Infinity, abortController.signal);
```

Comment:
Can you commit as comment the stack trace proving that this very specific event emitter gets more that 10 listeners?

Reply:
I could not create a situation where the warning is triggered locally. So I have removed the changes